test/csuite/wt4156_metadata_salvage/main.c

```diff
@@ -518,7 +518,7 @@ main(int argc, char *argv[])
      * We need to set up the string before we clean up the structure. Then after the clean up we
      * will run this command.
      */
-    testutil_check(__wt_snprintf(buf, sizeof(buf), "rm -rf core* %s*", home));
+    testutil_check(__wt_snprintf(buf, sizeof(buf), "rm -rf core* *.core %s*", home));
```

Comment:
Could there be a pattern of <prog>.core.<pid>? (Well there can be any pattern someone puts in the pattern file but none of that form at the moment.). Clearly `*core*` is too loose. But should the change generalize a little to `*.core*`?

I'm happy where it is because this is a trivial change and better than what we have and solves the problem at hand.